_posts/2022/2022-09-19-use-before-and-after-methods-when-comparing-dates-and-times-in-rails.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: article
+title: Use before? and after? Methods When Comparing Dates and Times in Rails
+description: I can’t remember which is greater than or less than either.
+category: ruby
+image:
+  base: 2022/use-before-and-after-methods-when-comparing-dates-and-times-in-rails
+  alt: Clocks piled on top of each other
+  source: https://unsplash.com/photos/FlHdnPO6dlw
+  credit: Jon Tyson
+---
+
+Active Support contains many additions to the basic classes that form the standard libraries of Ruby. There are extensions to `String`, `Hash`, `Array` and even `Date` and `Time`.
+
+If you’ve been working inside Rails you might not realise that these are Rails-isms rather than Ruby-isms.
```

Comment:
*If you spend most of your time working inside Rails, you might not realise...

(But also, doesn't this sentence only work the other way around, if they were in fact 'Ruby-isms'. Like wouldn't you assume they were Rails-isms...? Maybe not...)

Reply:
```suggestion
If you spend most of your time working inside Rails applications, you might not realise that these are Rails-isms rather than Ruby-isms.
```